client/galaxy/scripts/mvc/workflow/workflow-connector.js

```diff
@@ -1,16 +1,17 @@
 import $ from "jquery";
-
+import * as d3 from "d3";
```

Comment:
In general we want to get away from this and use specific imports `import {select} from ...", but our version of d3 is out of date and not worth fussing with right now since this is how we use it elsewhere; we can fix them all in a single pass.